src/System.CommandLine.Hosting/System.CommandLine.Hosting.csproj

```diff
@@ -11,9 +11,12 @@
     <DebugType>portable</DebugType>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="2.2.0" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.1'">
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="3.1.5" />
+  </ItemGroup>
```

Comment:
Multi-targeting package reference for .NET Standard 2.0 and 2.1.

I suggest that we will drop support for .NET Standard 2.0 at some point in the future for maintainability reasons. However, it might be prudent to temporarily support both TFMs in a transition period until .NET Core 2.1 is no longer covered by LTS support (scheduled for 2021-08-21) and .NET 5 (or maybe even .NET 6) has been firmly established.